### PR TITLE
Css practical

### PR DESCRIPTION
**Pull request by professor-Ibisi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>WEB CREATIONS</title>
+        <link rel="stylesheet" href = "style.css">
+    </head>
+    <body>
+
```
Comment (leonardk1) [inline index.html:3]: this space is not neccessary

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                    <li><a href ="*">about</a></li>
+                    <li><a href ="*">contact</a></li>
+                </ul>
+
+           </div>
+
+           <div class="title">
+               <h1> UBUNI-SHULE WEB CREATIONS</h1>
+           </div>
+
```
Comment (leonardk1) [inline index.html]: this space is not necessary

**Review by leonardk1 — APPROVED** [review]
make those changes I have suggested and merge

File: index.html (hunk 2 above)
Comment (professor-Ibisi) [reply index.html]: which one brother

Comment (professor-Ibisi) [reply index.html]: ok i have seen it

Comment (professor-Ibisi) [reply index.html]: i have changed it

Comment (professor-Ibisi) [reply index.html]: before i merge i want your last sugestion

File: box model.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>The box model</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
```
Comment (leonardk1) [inline box model.html:8]: space not needed

File: box model.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      surrounding that you have the border and surrounding that you have the margin.<br>
+      can be visually represented like this:</p>
+  </div>
+  <div class="a">
+    <p align="center">You don’t have to use all of these,<br>
+      but it can be helpful to remember that the box model <br> be applied to every element on the page,<br>
+      and that’s a powerful thing!</p>
+  </div>
+</body>
+
```
Comment (leonardk1) [inline box model.html:24]: space not needed

**Pull request merged**